hubble-be/src/main/java/com/baidu/hugegraph/service/load/LoadTaskService.java

```diff
@@ -364,7 +364,8 @@ private LoadOptions buildLoadOptions(GraphConnection connection,
         options.token = connection.getPassword();
         options.protocol = connection.getProtocol();
         options.trustStoreFile = connection.getTrustStoreFile();
-        options.trustStorePassword = connection.getTrustStorePassword();
+        // TODO: we need change it to token?
```

Comment:
it's for https

Reply:
but we miss the method in LoadOptions now (it was renamed?)

Reply:
maybe changed in: https://github.com/hugegraph/hugegraph-loader/pull/204/files

Reply:
I means the param `options.trustStorePassword` is renamed

![image](https://user-images.githubusercontent.com/17706099/147747463-755692e8-bb42-4203-8e76-a81bda722c04.png)